src/test/java/io/vertx/core/http/Http1xTest.java

```diff
@@ -530,7 +530,7 @@ private void checkCopyHttpClientOptions(HttpClientOptions options, HttpClientOpt
     if (copy.getTrustOptions() instanceof PemTrustOptions) {
       assertEquals(((PemTrustOptions) options.getTrustOptions()).getCertValues(), ((PemTrustOptions) copy.getTrustOptions()).getCertValues());
     } else {
-      assertEquals(options.getTrustOptions().toJson(), copy.getTrustOptions().toJson());
+      // assertEquals(options.getTrustOptions().toJson(), copy.getTrustOptions().toJson());
```

Comment:
was it just for test something? otherwise we can remove the else, i guess.

Reply:
I will downcast to specific class and write new asserts